packages/modules/init-capital/src/actions/subactions/tx-builders/DoubleSlopeIRM/deploy.ts

```diff
@@ -0,0 +1,45 @@
+import { Hex, encodeDeployData } from 'viem'
+
+import { InfinitWallet, TransactionData, TxBuilder } from '@infinit-xyz/core'
+
+import { readArtifact } from '@/src/utils/artifact'
+
+export interface DeployDoubleSlopeIRMTxBuilderParams {
+  baseBorrowRateE18: bigint
+  jumpUtilizationRateE18: bigint
+  borrowRateMultiplierE18: bigint
+  jumpRateMultiplierE18: bigint
+}
+
+export class DeployDoubleSlopeIRMTxBuilder extends TxBuilder {
+  baseBorrowRateE18: bigint
+  jumpUtilizationRateE18: bigint
+  borrowRateMultiplierE18: bigint
+  jumpRateMultiplierE18: bigint
+
+  constructor(client: InfinitWallet, params: DeployDoubleSlopeIRMTxBuilderParams) {
+    super(DeployDoubleSlopeIRMTxBuilder.name, client)
+    this.baseBorrowRateE18 = params.baseBorrowRateE18
+    this.jumpUtilizationRateE18 = params.jumpUtilizationRateE18
+    this.borrowRateMultiplierE18 = params.borrowRateMultiplierE18
+    this.jumpRateMultiplierE18 = params.jumpRateMultiplierE18
+  }
+
+  async buildTx(): Promise<TransactionData> {
+    const liqIncentiveCalculatorArtifact = await readArtifact('DoubleSlopeIRM')
```

Comment:
wrong variable name, should be
`const DoubleSlopeIRMArtifact = await readArtifact('DoubleSlopeIRM')`